client/app/components/blueprints/blueprint-editor/_blueprint-editor.sass

```diff
@@ -1,3 +1,6 @@
+.capitalize
```

Comment:
OH true, that doesn't exist (but maybe a filter we should add if this is something that has value elsewhere)